messaging/src/main/java/org/axonframework/eventhandling/scheduling/quartz/QuartzEventScheduler.java

```diff
@@ -305,7 +305,9 @@ private Object deserializePayload(JobDataMap jobDataMap) {
         }
 
         private Instant retrieveDeadlineTimestamp(JobDataMap jobDataMap) {
-            return Instant.ofEpochMilli((long) jobDataMap.get(MESSAGE_TIMESTAMP));
+            Object timestamp = jobDataMap.get(MESSAGE_TIMESTAMP);
+            if (timestamp instanceof String) return Instant.parse(timestamp.toString());
```

Comment:
Same argument as with the `DeadlineJob`.

Reply:
fixed in subsequent commit